libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/behandling/StatistikkBehandling.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.etterlatte.libs.common.behandling
+
+import no.nav.etterlatte.libs.common.Vedtaksloesning
+import no.nav.etterlatte.libs.common.sak.Sak
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class StatistikkBehandling(
```

Comment:
Er allerede et problem at det finnes to typer `DetaljertBehandling`, så det er fint om vi ikke får _en tredje variant_.

Reply:
Neida men refaktoreringen er i to deler som jeg sa til deg og detaljertbehandling skal bort fra statistikk og endres etterhvert da det ikke er behov for den lenger.

Reply:
Ah, ja, det har gått litt fort i lesingen. Eneste jeg er redd for er at dette blir enda en ting "vi skal rydde i fremtiden" som aldri blir gjort. Den listen er allerede _laaaang_. 😅